sdks/python/apache_beam/runners/dataflow/internal/dependency.py

```diff
@@ -513,7 +545,7 @@ def get_sdk_package_name():
   """For internal use only; no backwards-compatibility guarantees.
 
   Returns the PyPI package name to be staged to Google Cloud Dataflow."""
-  container_version = get_required_container_version()
+  container_version = _get_required_container_version()
```

Comment:
Let's separate the changes so we can revert them independently if needed, opened
https://github.com/apache/beam/pull/3473